src/pages/Rockets/RocketInfo.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reserveRocket, cancelRocket } from '../../redux/rocketSlice';
+
+const RocketInfo = ({
+  name, description, image, id, reserved,
+}) => {
+  const dispatch = useDispatch();
+
+  const toggleReservedStatus = () => {
+    if (reserved) {
+      dispatch(cancelRocket(id));
+    } else {
+      dispatch(reserveRocket(id));
+    }
+  };
+
+  const buttonClass = reserved ? 'reserved-button' : 'reserve-rocket';
+
+  return (
+    <div className="rockets-container">
+      <div>
+        <img className="rockets-pictures" src={image[0]} alt="rockets-pictures" />
+      </div>
+      <div className="rockets-info">
+        <h2 className="rocket-name">{name}</h2>
+        <p>
+          {reserved ? <span className="reservation-status">Reserved</span> : null}
+          {description}
+        </p>
+        <button
+          type="button"
+          className={buttonClass}
+          onClick={toggleReservedStatus}
+        >
+          {reserved ? 'Cancel Reservation' : 'Reserve Rocket'}
+        </button>
+      </div>
+    </div>
```

Comment:
👍 Great job on implementing the rocket info card! However, there's a small improvement I'd like to suggest for better HTML semantics:

- [ ] Please consider using the `<article>` tag instead of the `<div>` tag to represent the rocket info card. According to [HTML best practices](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article#:~:text=The%20%3Carticle%3E,item%20of%20content.), the `<article>` tag is more appropriate for this purpose. It helps enhance the semantic structure of your HTML and can improve accessibility.

By making this change, you'll align your code with recommended HTML practices, which can benefit both your code's semantics and maintainability. Keep up the great work! 👍🚀